RESOURCES.md

```diff
@@ -34,9 +38,13 @@ Below are links to various places you can find the Kentico developer community.
 
 - [Kentico Xperience DevNet](https://devnet.kentico.com/) - The central resource for the Kentico Xperience developer community
 
-- [Kentico Community Slack](https://kentico-community.slack.com/) - A Slack workspace open to Kentico Gold Partners, invite-only to everyone else. If you'd like to join, contact one of the Kentico developer advocates or MVPs [below on Twitter](#kentico-mvps)
+- [Kentico Community Slack](https://kentico-community.slack.com/) - A Slack workspace open to Kentico Gold Partners, invite-only to everyone else. If you'd like to join, contact one of the Kentico developer advocates or MVPs [below on Twitter](#kentico-mvps).
+
+  Technical discussions about, and developer advice for both Kontent and Experience can be found here.
+
+- [Kentico Kontent MVPs](https://kontent.ai/blog/mvp-announcement) - List of all Kentico Kontent MVPs
```

Comment:
I need the correct links for both the Kontent MVP page and Xperience MVP page.

Reply:
Here's the Xperience MVP page https://xperience.io/partnership/mvp-program